builtins/amp-img.js

```diff
@@ -188,18 +139,13 @@ export class AmpImg extends BaseElement {
   /** @override */
   layoutCallback() {
     this.initialize_();
-    let promise = this.updateImageSrc_();
```

Comment:
This logic block was refactored into the `load` and `error` listeners on the right.